src/create.ts

```diff
@@ -64,7 +65,7 @@ export const args: Record<ArgName, yargs.Options> = {
   },
   'type': {
     description: 'Type package do you want to develop',
-    choices: ['native', 'cpp', 'expo'],
+    choices: ['native', 'swift-native', 'cpp', 'expo'],
```

Comment:
Looks like a typo? Above it's `native-swift`

Reply:
Yes you're right. I just tested the project init with the wizard, so i didn't recognize this.